models/DataObject/Classificationstore.php

```diff
@@ -297,7 +301,11 @@ public function setFieldname($fieldname)
      */
     public function getActiveGroups()
     {
-        return $this->activeGroups;
+        $doGetInheritedValues = Model\DataObject::doGetInheritedValues();
```

Comment:
using DataObject:doGetInheritedValues(); to avoid breaking Changes

 has it is done in getLocalizedKeyValue()